src/Localization/Localization/src/Internal/AssemblyWrapper.cs

```diff
@@ -25,8 +25,8 @@ public AssemblyWrapper(Assembly assembly)
 
         public Assembly Assembly { get; }
 
-        public virtual string FullName => Assembly.FullName;
+        public virtual string FullName => Assembly.FullName!;
```

Comment:
> Or does this actually mean "Assembly.FullName CLAIMS that it will return string?, but I know better and I'm telling you that that value will always be a non-null string"?

This